### ISSUE TITLE
508-defect-2 [KEYBOARD]: Focus SHOULD be restored to the Add Another button when users press Remove when editing in place

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#13007]
## [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Focus is not being returned to the Add Another button when users press Remove on the Review > edit in place UI. Screen shot attached below.

Please advise if this is a forms library issue, and I'll reassign it to VSP Tools FE.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] As a keyboard user, I want to be able to add another dependent quickly, without having to tab multiple times after pressing Remove.

## Steps to Recreate

* Enter https://staging.va.gov/view-change-dependents/add-remove-form-686c/ in browser
* Check two or three dependent situations, including adding and removing at least one dependent each
* Navigate to the **Review** page and open the accordions
* Click "Edit" to enter inline editing mode, add a dependent, then click the "Remove" button after the generic card appears to add a dependent.
* Verify that focus is not returned to the Add Another button

## WCAG or Vendor Guidance (optional)

* [Focus Order: Understanding SC 2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)

## Screenshots or Trace Logs

![Screen Shot 2020-08-28 at 9 46 06 AM](https://user-images.githubusercontent.com/934879/91615379-08ef3400-e949-11ea-838a-88d08f991897.png)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by jason-gcio** [comment]
We are not equipped to edit this action of resetting the focus as it lies inside of the Form System.

**Comment by 1Copenut** [comment]
@jason-gcio I've passed this issue on to Carol and the design system team. I'll leave you as an assignee for now.

**Comment by jason-gcio** [comment]
Follow up on

**Comment by micahchiang** [comment]
Just tested this. Hasn't been resolved yet, but is platform work. @jason-gcio

**Comment by taylorkaren** [comment]
Forms Library Team Ticket: https://app.zenhub.com/workspaces/forms-library---platform-spike-team-61b0ae1f2cd3c30014e8a5b0/issues/department-of-veterans-affairs/formulate/89.

**Comment by micahchiang** [comment]
The Forms Library Team will be presiding over this ticket - here is the ticket for the forms library team - https://github.com/department-of-veterans-affairs/formulate/issues/89

**Comment by taylorkaren** [comment]
Closing this ticket as this issue will be addressed on VAFSC ticket https://github.com/department-of-veterans-affairs/va-forms-system-core/issues/89.
